src/main/java/racinggame/domain/RacingGame.java

```diff
@@ -0,0 +1,57 @@
+package racinggame.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+    private final List<Car> cars = new ArrayList<>();
+
+    public RacingGame(String names) {
+        splitCarName(names);
+    }
```

Comment:
```suggestion
    public RacingGame(String names) {
        this(createCars(names));
    }
    
    public RacingGame(List<Car> cars) {
        this.cars = cars;
    }
```
필드 타입과 같은 주 생성자를 추가하고, 부 생성자가 주 생성자를 호출하는 방식으로 구현하는 습관을 들인다.